src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/BlobAuditingTests.ps1

```diff
@@ -0,0 +1,834 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+Tests setting and getting blob auditing policy with classic storage
+#>
+function Test-BlobAuditingUpdatePolicyWithClassicStorage
+{
+	# Setup
+	$testSuffix = 10362
```

Comment:
Don't hardcode, it makes re-recording impossible if some other subscription creates a server with that name. Instead use getAssetName. See AuditingPolicyTests.ps1